datafusion/proto/src/common.rs

```diff
@@ -17,26 +17,17 @@
 
 use datafusion_common::{DataFusionError, Result};
 
-pub fn csv_delimiter_to_string(b: u8) -> Result<String> {
-    let b = &[b];
-    let b = std::str::from_utf8(b)
-        .map_err(|_| DataFusionError::Internal("Invalid CSV delimiter".to_owned()))?;
-    Ok(b.to_owned())
-}
-
-pub fn str_to_byte(s: &String) -> Result<u8> {
+pub fn str_to_byte(s: &String, flag: &str) -> Result<u8> {
     if s.len() != 1 {
-        return Err(DataFusionError::Internal(
-            "Invalid CSV delimiter".to_owned(),
-        ));
+        return Err(DataFusionError::Internal(format!("Invalid CSV {flag}")));
     }
     Ok(s.as_bytes()[0])
 }
 
-pub fn byte_to_string(b: u8) -> Result<String> {
+pub fn byte_to_string(b: u8, flag: &str) -> Result<String> {
```

Comment:
```suggestion
fn byte_to_string(b: u8, flag: &str) -> Result<String> {
```

Reply:
I am confuing about this, `str_to_bytes` is called in other file.